src/ui/Templates/TemplateCache.ts

```diff
@@ -79,6 +80,10 @@ export class TemplateCache {
     return _.keys(TemplateCache.defaultTemplates);
   }
 
+  /**
+   * Get a default template by name.
+   * @param name The name of the queried template
```

Comment:
I'm seeing that those Template module are not tested properly (TemplateCache, HTMLTemplate, UnderscoreTemplate, etc).

That's not good in itself. 

But, if we want to document those methods, we should ensure we have some UT so that we do not break the documented API by mistake. Because then that's even worse :)